src/index.js

```diff
@@ -107,6 +115,16 @@ export default class AuthnWidget {
     this.addPostRenderCallback('EXTERNAL_AUTHENTICATION_FAILED', this.externalAuthnFailure);
     this.addEventHandler('DEVICE_PROFILE_REQUIRED', this.postDeviceProfileAction);
     this.addEventHandler('REFERENCE_ID_REQUIRED', this.registerAgentlessHandler);
+    this.addPostRenderCallback('AUTHENTICATION_REQUIRED', this.postEmptyAuthentication);
+    this.addPostRenderCallback('MFA_COMPLETED', this.postContinueAuthentication);
+    this.addEventHandler('DEVICE_SELECTION_REQUIRED', this.registerMfaEventHandler);
+    this.addEventHandler('OTP_REQUIRED', this.registerMfaEventHandler);
+    this.addEventHandler('OTP_REQUIRED', this.registerMfaChangeDeviceEventHandler);
+    this.addPostRenderCallback('PUSH_CONFIRMATION_WAITING', this.postPushNotificationWait);
+    this.addEventHandler('PUSH_CONFIRMATION_WAITING', this.registerMfaChangeDeviceEventHandler);
+    this.addEventHandler('PUSH_CONFIRMATION_TIMED_OUT', this.registerMfaEventHandler);
+    this.addEventHandler('PUSH_CONFIRMATION_TIMED_OUT', this.registerMfaChangeDeviceEventHandler);
+    this.addPostRenderCallback('MOBILE_PAIRING_REQUIRED', this.postContinueAuthentication);
```

Comment:
Curious why `MOBILE_PAIRING_REQUIRED` is not a part of CORE_STATES?

Reply:
An event handler is not required in that state state, I suppose I should be removing `AUTHENTICATION_REQUIRED` from the core states as well since it's the same case.